pycbc/io/live.py

```diff
@@ -216,17 +226,22 @@ def __init__(self, ifos, coinc_results, **kwargs):
 
         # P astro calculation
         if 'padata' in kwargs:
-            padata = kwargs['padata']
+            assert len(coinc_ifos) < 3, \
+                f"p_astro can't handle {coinc_ifos} coinc ifos!"
             trigger_data = {
                 'mass1': sngl_populated.mass1,
                 'mass2': sngl_populated.mass2,
                 'spin1z': sngl_populated.spin1z,
                 'spin2z': sngl_populated.spin2z,
                 'network_snr': network_snrsq ** 0.5,
-                'far': far}
-            horizons = {ifo: self.psds[ifo].dist for ifo in self.psds}
+                'far': far,
+                'triggered': coinc_ifos,
+                # Active = potentially contributing to significance:
+                # ignore ifos that only contribute to sky loc
```

Comment:
I would expect "active" to mean "with usable data", and I think in other contexts we do indeed use "active" to mean "in observing mode", so I am slightly concerned about this nomenclature here. Why not "trig_ifos" / "sig_ifos" / "ifos"?

Reply:
You mean the use of 'active' in the trigger_data dict that is fed to pastro? The thinking there is, the ifos active from the point of view of possibly influencing p astro.  The p astro module does not need to know about or think about other ifos ;)

Reply:
Has been swapped out for 'sensitive'